scimono-compliance-tests/src/main/java/com/sap/scimono/scim/system/tests/launcher/LauncherProperties.java

```diff
@@ -101,18 +101,19 @@ public static LauncherProperties fromCommandLineArguments(String[] args) {
     for (String arg : args) {
       String[] propertyNameValuePair = arg.split(PROPERTY_NAME_VALUE_DELIMITER);
       ParamName propertyName = ParamName.fromPropertyName(propertyNameValuePair[0]);
-      String propertyValue = propertyNameValuePair[1];
 
       if(propertyName == null) {
         continue;
       }
 
+      String propertyValue = arg.substring(propertyName.value().length() + 1);
```

Comment:
You don't need this. The split method can accept second argument for the desired number of partitions starting from the beginning